bulletroboy/operator.py

```diff
@@ -42,6 +42,29 @@ def get_link_index(self, link_name):
 	def move(self, case):
 		self.movements.simple_move(case)
 
+	def publish_state(self, publisher, msg):
```

Comment:
also the end effectors must be passed as parameter to this function as an array

Reply:
```# is it appropriate to define the endeffector array in init?
   def __init__():
         ...
         self.endEffectors = np.array(['human/left_hand','human/right_hand'])

   def publish_state(self, publisher, msg, self.endEffectors):
```

Reply:
pass them as parameters and assign default values

Reply:
if you have self.endeffectors, you can access it inside of the member function without the need to pass it as a parameter

Reply:
the endeffector link_names (ef_names) are now passed as a parameter in the publish_state function with the default being: np.array(['human/left_hand','human/right_hand'])

      def publish_state(self, publisher, msg, ef_names = np.array(['human/left_hand','human/right_hand']))